ext/spl/spl_directory.c

```diff
@@ -2125,14 +2125,24 @@ static int spl_filesystem_file_call(spl_filesystem_object *intern, zend_function
 	spl_filesystem_file_call(intern, func_ptr, pass_num_args, return_value, arg2); \
 } /* }}} */
 
+static int is_csv_line_empty(spl_filesystem_object *intern) /* {{{ */
```

Comment:
```suggestion
static zend_bool is_line_empty(spl_filesystem_object *intern) /* {{{ */
```
Method not really CSV specific?

Reply:
Right! I've renamed the function.